code/go/0chain.net/blobber/main.go

```diff
@@ -172,31 +172,31 @@ func checkForDBConnection() {
 func processMinioConfig(reader io.Reader) error {
```

Comment:
there is probably a better way of doing this rather than having to check `scanner.Scan()` after each line?